libmamba/tests/repodata_json_cache/test_7.state.json

```diff
@@ -11,5 +11,7 @@
         "value": true
     },
     "mod": "Fri, 11 Feb 2022 13:52:44 GMT",
+    "mtime_ns": -4762331256009361200,
```

Comment:
maybe remove this? We should add this to the .gitignore files nad figure out why the `mtime_ns` is negative on linux.